README.md

```diff
@@ -13,6 +13,55 @@ A distributed task manager.
 
 Real-world use cases for Kōjō and instructions for getting Kōjō up and running in your project can be found at [KojoFitness](https://github.com/neighborhoods/KojoFitness).
 
+## Configuring Kōjō
+The `kojo_job_table` database table and the `\Neighborhoods\Kojo\Api\V1\Job\Type\ServiceInterface::getNewJobTypeRegistrar` Kōjō API client provide ways to control the scheduling behavior of Kōjō workers.  
+
+Each column in the `kojo_job_table` has a specific purpose
+- `type_code`: machine_readable_name used to select and schedule jobs. Typical convention is to use all lowercase with underscores snake_case.
+- `name`: Human-readable name which can be used for display purposes
+- `worker_uri`: Class which will be created for the job
+- `worker_method`: Method of `worker_uri` class that is called when a job is run
+- `can_work_in_parallel`: Defines if multiple instances of job can be worked at the same time across all execution environments
+- `default_importance`:  Defines the priority of the job. Jobs with a higher importance will get scheduled first when resources are constrained
+- `cron_expression`: [crontab](https://crontab.guru/) formatted expression of how often the job will be added to the schedule, or null if the job is supposed to be dynamically scheduled
+- `schedule_limit`:  Number of concurrent jobs allowed to be in a `working` state at the same time.
+- `schedule_limit_allowance`: Number of jobs allowed to be in a `waiting` state at the same time.
+- `is_enabled`: Turn the job on and off.
```

Comment:
I might add something here about how this affects statically- vs dynamically-scheduled jobs differently. For the former, it won't schedule any more, but if anything was already scheduled, it'll still be worked. For the latter, you'll get an exception from the WorkerService's job scheduler if you try to schedule a disabled job type